Drift/Private/DriftBase.cpp

```diff
@@ -2625,6 +2625,9 @@ void FDriftBase::GetUserInfo()
             return;
         }
 
+        const auto PlayerUuid = currentUser.FindField(TEXT("player_uuid"));
```

Comment:
I think it would be prudent to account for this field potentially being missing since it's conceivable that this module would be used in a product running against a older version of drift-base